boards/raspberry_pi_pico/src/test/pwm.rs

```diff
@@ -0,0 +1,231 @@
+//! Integration tests for PWM peripheral
```

Comment:
Good idea to have tests for the implementation!
However, if the tests that you created do not depend on the board as a whole and just test the pwm implementation, I would move them to the chip/rp2040 folder. To achieve this, either create a subfolder in chips/rp2040/src, or combine the test with the actual implementation.

Reply:
Done it. Remove most of the integration tests that didn't rely on the PWM HIL.

Reply:
@valexandru does this solve this issue?

Reply:
yes